editors/sced/sced/WindowHelper.py

```diff
@@ -49,9 +49,6 @@
         <menuitem action="ScedStartServer"/>
         <menuitem action="ScedStopServer"/>
         <separator/>
-        <menuitem action="ScedStartSwingOSC"/>
-        <menuitem action="ScedStopSwingOSC"/>
-        <separator/>
         <menuitem action="ScedFindHelp"/>
```

Comment:
I wonder what others think about updating editors here? Pro: These commands won't work at all after these changes. Con: User-facing, might annoy some people. (But, I don't know who is using sced anymore.)

Reply:
I don't even know what sced is. Seems unreasonable to keep stuff that has no function anymore? Maybe add it to some changelog then but I also never heard of SwingOSC before this PR, just trying to get more familliar with SC through these PR.

Reply:
Sced is a gedit plugin to run SC. It isn't popular. I'd guess SC-IDE is most widely used, then vim, then perhaps Atom or Sublime Text or Emacs. It probably won't impact any users to remove these -- just checking to see if anyone reading this has a different opinion.